_pages/community.html

```diff
@@ -143,3 +143,23 @@ <h5 class="mb-1">Join mailing list &raquo;</h5>
         </div>
     </div>
 </div>
+
+
+<!-- Chat Modal -->
+<div class="modal fade" id="chatModal" tabindex="-1" role="dialog" aria-labelledby="chatModalLabel" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="chatModalLabel">Chat with CompSoc</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+
+            <div class="modal-body">
+                {% capture modal %}true{% endcapture %}
+                {% include chat.html modal=modal %}
```

Comment:
Add a permalink from the modal to the page?

Reply:
the "Chat" button is a hyperlink to the chat page, so I'm hoping people would do `right-click -> copy link`.

Alternatively we could do [this](https://stackoverflow.com/a/3354511)?

Reply:
Or just a very small "permalink" link at the bottom of the modal? For sharing on the online services and such, otherwise it is pretty unnecessary.

Reply:
Eh let's just merge this.